packages/client/example.js

```diff
@@ -4,64 +4,40 @@
 
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
 
-const {client, xml} = require('./index') // For you; require('@xmpp/client')
+const {xml, client} = require('./index') // For you; require('@xmpp/client')
 const entity = client()
 
-// Emitted for any error
+// Log errors
 entity.on('error', err => {
-  console.error('error', err)
+  console.error('❌', err.toString())
 })
 
-entity.on('close', () => {
-  console.log('closed')
+// Log status changes
+entity.on('status', (status, value) => {
+  console.log('🛈', status, value ? value.toString() : '')
```

Comment:
status is always a string but value can be anything (a jid for `online` event for example)